src/structures/Guild.js

```diff
@@ -1422,6 +1470,45 @@ class Guild extends Base {
       .then(() => this);
   }
 
+  /**
+   * A `Partial` object is a representation of any existing object.
+   * This object contains between 1 and all of the original objects parameters.
```

Comment:
Since you actually can edit with an empty object.
```suggestion
   * This object contains between 0 and all of the original objects parameters.
```